Src/ILGPU.Tests.CPU/ILGPU.Tests.CPU.csproj

```diff
@@ -34,19 +39,16 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Src\ILGPU\ILGPU.csproj" />
-    <ProjectReference Include="..\ILGPU.Tests\ILGPU.Tests.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Update="Configurations.tt">
+    <None Remove="Configurations.tt" />
```

Comment:
I wonder about these `Remove` and `Include` operations. I suppose this is required to create the T4 templates automatically, right?

Reply:
Ah so this was needed when I first used Microsoft.TextTemplating.targets to run the transforms as that only worked with Content items, but now I've switched to using dotnet-t4 (because the first approach doesn't work with the dotnet commands) I realise I can revert this change and keep these as None items, so I'll do that to keep things simpler.

The tt files are automatically included as "None" items so this was removing them from the None items and then adding them as Content items.